index.html

```diff
@@ -2915,6 +2980,12 @@ <h2>Incremental Transfer</h2>
                 The HTTP/1.1 chunked transfer encoding is not supported in HTTP/2.
                 <!-- </span> -->
                 </li>
+                <li>
+        <!-- <span class="rfc2119-assertion" id="perf-inc-chunked-3"> -->
```

Comment:
we don't currently define CoAP-based directories (these are purely HTTP) put putting this here but commented out is helpful in case we ever do want to extend directories